types/nearley/index.d.ts

```diff
@@ -47,26 +49,46 @@ export interface ParserOptions {
 }
 
 export class Rule {
-    constructor(name: any, symbols: any, postprocess: any);
-    toString(withCursorAt: any): any;
     static highestId: number;
+
+    id: number;
+    name: string;
+    symbols: any[];
+    postprocess?(data: any[], reference: number, wantedBy: any): void;
```

Comment:
I'm not sure why `wantedBy` is here: the argument is always the value `Parser.fail`, which is typed as `static fail: {};` above. Looks like a typo to me. :)